lib/mapper.js

```diff
@@ -14,56 +14,31 @@ var each = require('@ndhoule/each');
 
 exports.track = function (msg, settings){
   var events = settings.events;
-  var conversionLabel = exports.getConversionLabel(msg.event(), events)
+  var whitelistedRemarketingEvent = exports.getRemarketingEvent(msg.event(), settings.whitelist);
```

Comment:
to match "events" & "remarketing" style, i suggest declaring and passing:
var whitelist = settings.whitelist;